docs/config.rst

```diff
@@ -194,6 +194,15 @@ Features
 
    See also: :attr:`LP_MARK_DISABLED`.
 
+.. attribute:: LP_ENABLE_AWS_PROFILE
+   :type: bool
+   :value: 1
+
+   Display the current value of the ``AWS_PROFILE`` or ``AWS_DEFAULT_PROFILE``
```

Comment:
Prefix these with `:envvar:`, like this:
```suggestion
   Display the current value of the :envvar:`AWS_PROFILE` or
   :envvar:`AWS_DEFAULT_PROFILE`.
```

It won't display any differently, but it will index and search better.

Reply:
Done.